source/faq.txt

```diff
@@ -79,10 +94,10 @@ Can I use polymorphism in a POJO accessor?
 
 Yes, by using a discriminator.
 
-What is the discriminator?
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+What is a discriminator?
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-The discriminator is for cases where you want to use inheritance, and
+A discriminator is for cases where you want to use inheritance, and
```

Comment:
Not necessary to change in this PR, but I noticed that this is passive voice ("A discriminator is [used] for cases...") and only describes what it is used for rather than what it is.
Could you create a JIRA ticket to address this?

Reply:
Here's the ticket: https://jira.mongodb.org/browse/DOCSP-17445